polynote-frontend/polynote/ui.js

```diff
@@ -761,7 +761,7 @@ export class NotebookUI {
 
         this.cellUI.addEventListener('InsertCellAfter', evt => {
            const current = this.cellUI.cells[evt.detail.cellId] || this.cellUI.cells[this.cellUI.el.querySelector('.cell-container').id];
-           const nextId = "Cell" + this.cellUI.cellCount;
```

Comment:
@jeremyrsmith As far as I could tell this looked like the only change that needed to be done - not sure if you had anything else in mind when you said

> I think the `Cell` classes need to be updated to deal with the numeric ID (and how to map that to an `id` attribute for their container element). 

I wasn't able to find any other place where we explicitly do stringy things to the cell id but I definitely could've missed something.

Reply:
Hmm... I guess the only other issue is that the ID passed to the cell constructor also ends up as a DOM element ID. For example now we'll have something like `<div id="1">` in the DOM. This isn't a legal `id` in HTML 4, though I guess it now is in HTML 5... but still doesn't seem like a good idea.

Reply:
ok! restored previous html ids :)